inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/convolution_negative.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include "shared_test_classes/single_layer/convolution.hpp"
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/test_constants.hpp"
+#include "../skip_tests_check.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32,
+        InferenceEngine::Precision::FP16
+};
+
+/* ============= 2D Convolution ============= */
+const std::vector<std::vector<size_t >> kernels2D = {
+    {1, 3},
+    {7, 1},
+    {3, 3},
+};
+
+const std::vector<std::vector<size_t >> InvalidKernels2D = {
+    {1, 4},
+    {2, 3},
+    {3, 2},
+    {8, 1},
+    {4, 4},
+};
+
+const std::vector<std::vector<size_t >> strides2D = {
+                                                          {1, 1},
+};
+const std::vector<std::vector<size_t >> strides2DInvalid = {
+                                                          {4, 4}, {1, 4}
+};
+const std::vector<std::vector<ptrdiff_t>> padBegins2D = { {0, 0},
+};
+const std::vector<std::vector<ptrdiff_t>> padEnds2D = { {0, 0},
+};
+const std::vector<std::vector<ptrdiff_t>> padBegins2DInvalid = { {1, 0}, {1, 1}, {0, 1}
+};
+const std::vector<std::vector<ptrdiff_t>> padEnds2DInvalid = { {1, 0}, {1, 1}, {0, 1}
+};
+const std::vector<std::vector<size_t >> dilations2D = { {1, 1},
+};
+const std::vector<std::vector<size_t >> dilations2DInvalid = { {2, 2},
+};
+const std::vector<size_t> numOutCannels2D = { 32 };
```

Comment:
Typo: should be Channels. Check in other places.

Reply:
I will improve with the next PR